src/rules/converters/variable-name.ts

```diff
@@ -0,0 +1,116 @@
+import { RuleConverter } from "../converter";
+
+export const convertVariableName: RuleConverter = tslintRule => {
+    const getCamelCaseRuleOptions = () => {
+        const camelCaseOptionNotice: string[] = [];
+        camelCaseOptionNotice.push(
+            "By default, this rule looks for any underscores (_) located within the source code. It ignores leading and trailing underscores and only checks those in the middle of a variable name.",
+        );
+
+        if (
+            tslintRule.ruleArguments.length !== 0 &&
+            tslintRule.ruleArguments.includes("require-const-for-all-caps")
+        ) {
+            camelCaseOptionNotice.push(
+                'The argument "require-const-for-all-caps" is not needed as ESlint will decide if a variable is a constant (all uppercase). If not, a warning will be thrown.',
+            );
+        }
+
+        if (
+            tslintRule.ruleArguments.length !== 0 &&
+            (tslintRule.ruleArguments.includes("allow-pascal-case") ||
+                tslintRule.ruleArguments.includes("allow-snake-case"))
+        ) {
+            camelCaseOptionNotice.push(
+                "This rule does not allow pascal neither snake case to variable names. Those are reserved for class names and static methods.",
+            );
+        }
+
+        return {
+            arguments: [],
+            notices: camelCaseOptionNotice,
+        };
+    };
+
+    const getUnderscoreDangleRuleOptions = () => {
+        const underscoreDangleOptionArguments: string[] = [];
+        const underscoreDangleOptionNotice: string[] = [];
+        underscoreDangleOptionNotice.push(
```

Comment:
Instead of describing how the rule options have changed, you'll need to change this to describe how the end behavior has changed. See the other notice arrays for similar suggestions.